packages/cli/tests/commands/start.spec.ts

```diff
@@ -14,18 +15,87 @@ vi.mock('execa', () => ({
   __esModule: true,
 }));
 
+vi.mock('../../src/lib/project-config', () => ({
+  getProjectConfig: vi.fn(() => ({
+    get: vi.fn((key) => {
+      if (key === 'framework') {
+        return 'catalyst';
+      }
+
+      return undefined;
+    }),
+  })),
+}));
+
+beforeAll(() => {
+  consola.wrapAll();
+});
+
+beforeEach(() => {
+  consola.mockTypes(() => vi.fn());
+});
+
+afterEach(() => {
+  vi.clearAllMocks();
+});
+
 test('properly configured Command instance', () => {
   expect(start).toBeInstanceOf(Command);
   expect(start.name()).toBe('start');
   expect(start.description()).toBe('Start your Catalyst storefront in optimized production mode.');
+  expect(start.options).toEqual(
+    expect.arrayContaining([
+      expect.objectContaining({
+        flags: '--framework <framework>',
+        argChoices: ['catalyst', 'nextjs'],
+      }),
```

Comment:
🍹 Again not for this PR, we should agree on a pattern. For example, I used `long` instead of `flags` and `argChoices`. 

Separately, I still wonder if these tests are worth writing, doesn't Commander kind of guarantee that options are attached to the Command as part of their own internal tests? (https://github.com/tj/commander.js/blob/master/tests/command.parseOptions.test.js)